### DISCUSSION TITLE
What is the definition of the priority labels (P0-P4) on OpenThread?

### DISCUSSION THREAD

**Discussion by tpmanley** [openthread/openthread#10808] [General]
I am wondering how the P0-P4 labels get assigned to OpenThread PRs. Is there any definition on what constitutes a P0, P1, P2, etc...? Is P0 the highest or lowest? I'm used to P0 being the most critical and P4 being very minor, but I've seen some that I think are pretty important that are P4s and some that seem much less important that are P2s so I'm just trying to get some clarity.

**Comment by abtink** [comment]
@tpmanley, P0 is the most critical, and P4 is minor. The priority (P0-P4) is often set based on the judgment of the developer who submits the pull request and/or suggestions from other team members.

If you have any suggestions for a particular pull request, please feel free to comment on the pull request to suggest other labels.

The `bug` label indicates a bug (often recommended as a candidate for cherry-picking into older firmware), whereas the `enhancement` label indicates an improvement , new feature, or possibly a fix to a non-critical issue.
